newIDE/app/src/ObjectEditor/Editors/SpriteEditor/CollisionMasksEditor/index.js

```diff
@@ -75,6 +76,10 @@ const CollisionMasksEditor = (props: Props) => {
     spriteIndex
   );
 
+  const objectHasPlatformerBehavior = props.object.hasBehaviorNamed(
```

Comment:
Slightly more complicated, but to be very precise we would need to iterate on behaviors and see if any of them has a type which is `PlatformBehavior::PlatformerObjectBehavior`.

This can be done with `object.getAllBehaviorNames().toJSArray()` and then iterate and for each behavior, get it (`getBehavior`) and on the resulting object uses `getTypeName`.